app/views/series/_form.html.erb

```diff
@@ -11,10 +11,12 @@
         </div>
     <% end %>
 
-    <div class="field">
-      Name<br/>
-      <%= @series.name %>
-    </div>
+    <% if request.path =~ /new/ %>
+      <div class="field">
+        <%= f.label :name, 'Name' %><br/>
+        <%= f.text_field :name, :size => 40 %>
```

Comment:
Since so much logic depends on the series mnemonic, we should have a more complicated tool to pick new names. (something like a text_field for the prefix component and select boxes for geography and frequency components). We should also be checking for collision with existing series.